modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -11,9 +11,9 @@
       source  => 'puppet:///modules/ocf_desktop/xsession/Xsession',
       require => File['/opt/share/xsession'];
     # printing and other notification script daemon
-    '/opt/share/puppet/notify.sh':
+    '/opt/share/puppet/notify':
       mode    => '0755',
-      source  => 'puppet:///modules/ocf_desktop/xsession/notify.sh';
+      source  => 'puppet:///modules/ocf_desktop/xsession/notify';
```

Comment:
We'll be installing stretch on the desktops soon enough, so I don't think it really matters.